server/form-pages/apply/move-on/placementDuration.ts

```diff
@@ -71,6 +88,18 @@ export default class PlacementDuration implements TasklistPage {
     return errors
   }
 
+  private lengthInDays(): number | null {
+    if (this.body.differentDuration === 'yes') {
+      if (this.body.durationDays && this.body.durationWeeks) {
```

Comment:
So if someone only entered weeks OR days would this not convert them? but that's not possible because of the validation

Reply:
Yeah, they should put a zero if it's an exact number of weeks or days